docs/src/dev/provider/gremlin-semantics.asciidoc

```diff
@@ -611,3 +611,84 @@ resolve to a `Map`.
 
 See: link:https://github.com/apache/tinkerpop/tree/master/gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/step/map/MergeVertexStep.java[source],
 link:https://tinkerpop.apache.org/docs/current/reference/#mergev-step[reference]
+
+=== call()
```

Comment:
nice - thanks for adding to this section of docs. could you please move it above `mergeV/E()` to hold alphabetical order? also, i'm not sure I recall what i was thinking when i developed the style for this documentation, but i pointed the docs at `master` and `current` in the links - i would think those should be switched to the documentation system's dynamic form of x.y.z. For example:

https://tinkerpop.apache.org/docs/x.y.z/reference/#call-step

https://github.com/apache/tinkerpop/tree/x.y.z/gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/structure/service/Service.java

The documentation system will replace that "x.y.z" variable with the current version on generation. If you don't mind, please make the changes to `mergeV/E()` as well.